examples/image-to-text/README.md

```diff
@@ -125,11 +125,50 @@ QUANT_CONFIG=./quantization_config/maxabs_measure.json python run_pipeline.py \
 --bf16
 ```
 
-Here is an example to quantize the model based on previous measurements for Llava-v1.6-mistral-7b-hf:
+Here is an example to quantize the model based on previous measurements for Llava-v1.6-mistral-7b:
 ```bash
 QUANT_CONFIG=./quantization_config/maxabs_quant.json python run_pipeline.py \
 --model_name_or_path llava-hf/llava-v1.6-vicuna-13b-hf \
 --image_path "https://llava-vl.github.io/static/images/view.jpg" \
 --use_hpu_graphs \
 --bf16
 ```
+
+### Inference with FusedSDPA
+
+More information on Using Fused Scaled Dot Product Attention (FusedSDPA) here:
```

Comment:
Habana FusedSDPA is a fused and optimized implementation of torch.nn.functional.scaled_dot_product_attention() for Gaudi. For more details, refer to [Habana online document](https://docs.habana.ai/en/latest/PyTorch/Model_Optimization_PyTorch/Optimization_in_PyTorch_Models.html?highlight=fusedsdpa#using-fused-scaled-dot-product-attention-fusedsdpa).